packages/web-forms/src/components/controls/Range/RangeSlider.vue

```diff
@@ -17,27 +22,37 @@ interface SliderDualValueProps extends PrimeSliderProps {
 	range: true;
 }
 
-interface SliderSingleValueEmits extends PrimeSliderEmits {
```

Comment:
This was one of the tricky ones to fix and perhaps something I would try not to do in other components unless really necessary. The `PrimeSliderEmits` doesn't exist anymore, and I found these equivalent types. It doesn't affect runtime but type-check